exercises/anagram/anagram_test.py

```diff
@@ -1,13 +1,13 @@
-# Python 2/3 compatibility
-if not hasattr(unittest.TestCase, "assertCountEqual"):
-    unittest.TestCase.assertCountEqual = unittest.TestCase.assertItemsEqual
-
 import unittest
 
 from anagram import find_anagrams
 
 # Tests adapted from `problem-specifications//canonical-data.json` @ v1.5.0
 
+# Python 2/3 compatibility
+if not hasattr(unittest.TestCase, "assertCountEqual"):
+    unittest.TestCase.assertCountEqual = unittest.TestCase.assertItemsEqual
```

Comment:
I'm confused: why is this needed? `assertCountEqual` is not referenced in this file.

Reply:
It's mentioned in the non-template test file, in the test for two and three anagrams, which happen to be the only tests that return multiple anagrams.

I don't believe we've got any requirement for the output to be sorted and _maybe_ someone wants to parallelize?

Reply:
> I'm confused: why is this needed? assertCountEqual is not referenced in this file.
I was confused about the need of this portion of code as well which is I removed it in the first place. 

> I don't believe we've got any requirement for the output to be sorted and maybe someone wants to parallelize?
In that case, is it better to replace `assertEqual` with `assertItemsEqual` in the template and this compatibility code can be removed as well?

Reply:
Well no, we’d need to keep the compatibility code and convert assertEqual to assertCountEqual, since assertItemsEqual was removed in 3.